src/test/java/org/fcrepo/importexport/exporter/ExporterTest.java

```diff
@@ -145,6 +147,15 @@ public void setUp() throws Exception {
         when(headResponse.getStatusCode()).thenReturn(200);
     }
 
+    @After
+    public void tearDown() {
+        try {
+            FileUtils.deleteDirectory(new File(exportDirectory));
```

Comment:
Some unit tests were failing when this wasn't in place.  Downstream tests check for the absence of certain files.